src/OSPSuite.UI/Views/BaseExplorerView.cs

```diff
@@ -129,9 +129,15 @@ public void Display()
       public virtual ITreeNode AddNode(ITreeNode nodeToAdd)
       {
          treeView.AddNode(nodeToAdd);
+         AdjustTreeViewColumns();
```

Comment:
When adding a node, it adjuste the size visible to see everything=> Scroll bar will be visible